### PR TITLE
Move from SETEX to SET EX

### PR DESCRIPTION
**Pull request by solracsf**
## Summary

SETEX is depreacted and may be removed: https://redis.io/commands/setex/
PHPredis handles the redirection if needed: https://github.com/phpredis/phpredis#set

 (+) small code optimizations

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

**Review by come-nc — COMMENTED** [review]
Please avoid mixing changes and refactoring in a single commit like this, it took me a long time to spot the actual change.
Mixing them in a same PR is fine but at least split the commits.

**Comment by solracsf** [comment]
> Please avoid mixing changes and refactoring in a single commit like this, it took me a long time to spot the actual change. Mixing them in a same PR is fine but at least split the commits.

Noted.

File: lib/private/Memcache/Redis.php [hunk 1]
```diff
@@ -53,32 +53,29 @@ public function getCache() {
...
-		$result = $this->getCache()->get($this->getPrefix() . $key);
-		if ($result === false && !$this->getCache()->exists($this->getPrefix() . $key)) {
+		$key = $this->getPrefix() . $key;
+		$result = $this->getCache()->get($key);
+		if ($result === false && !$this->getCache()->exists($key)) {
 			return null;
-		} else {
-			return json_decode($result, true);
 		}
+		return json_decode($result, true);
```
[the remark this answers is not in the document]
Comment (solracsf) [reply lib/private/Memcache/Redis.php:61]: Makes sense, but call to `exists($key)` was already there. Another opinion?

[the remark this answers is not in the document]
Comment (solracsf) [reply lib/private/Memcache/Redis.php:61]: Fixed

**Pull request closed without merging**